src/bindings/js/node/tests/infer_request.test.js

```diff
@@ -68,15 +69,43 @@ describe('InferRequest', () => {
     it(`Test infer([data]) throws ${msg}`, () => {
       assert.throws(
         () => inferRequest.infer([tl]),
-        {message: msg});
+        {message: new RegExp(msg)});
     });
     it(`Test infer({ data: tl}) throws ${msg}`, () => {
       assert.throws(
         () => inferRequest.infer({data: tl}),
-        {message: msg});
+        {message: new RegExp(msg)});
     });
   });
 
+  it('Test inferAsync(inputData: { [inputName: string]: Tensor })', () => {
+    inferRequestAsync.inferAsync({ data: tensor }).then(result => {
```

Comment:
Why don't use same **inferRequest** variable for async inference?

Reply:
It will be not safe. Synchronous `infer()` assumes that **inferRequest** object is not shared between threads.